backend/src/utils/logging/logger.ts

```diff
@@ -0,0 +1,24 @@
+import pino from "pino";
+
+export const logger = pino({
+  level: process.env.PINO_LOG_LEVEL || "trace",
+  timestamp: pino.stdTimeFunctions.isoTime,
+  formatters: {
+    bindings: (bindings) => {
+        return {
+            pid: bindings.pid,
+            hostname: bindings.hostname
+            // node_version: process.version
+        };
+    },
+  },
+  transport: {
+    targets: [
+      {
+        target: "pino-pretty",
```

Comment:
Pino pretty is only needed in dev mode. 

On production it will slow down pino, rather just transport it and analyze it somewhere